azure/services/virtualmachines/virtualmachines.go

```diff
@@ -468,6 +496,20 @@ func (s *Service) generateOSProfile(ctx context.Context, vmSpec azure.VMSpec) (*
 		osProfile.AdminPassword = to.StringPtr(generators.SudoRandomPassword(123))
 		osProfile.WindowsConfiguration = &compute.WindowsConfiguration{
 			EnableAutomaticUpdates: to.BoolPtr(false),
+			AdditionalUnattendContent: &[]compute.AdditionalUnattendContent{
```

Comment:
Perhaps, this would be better exposed through an API, rather than statically defining.